cmd/heimdall-identity-io/heimdall_identity_io.go

```diff
@@ -136,7 +136,7 @@ func (s *service) Init(ctx context.Context, cancel context.CancelFunc) {
 		return []string{s.accounts.PublicKey()}
 	}))
 	s.hashtagStatistics = hashtagstatistics.New(ctx)
-	s.nftContent = nftcontent.New(ctx, s.accounts, ionIndexer)
+	s.nftContent = nftcontent.New(ctx, s.accounts, ionIndexer, s.tokenAnalytics)
```

Comment:
make sure u maintain the logic @ice-dionysos added that disables token analytics for prod